### DISCUSSION TITLE
✨ New Website

### DISCUSSION THREAD

**Discussion by caarlos0** [goreleaser/goreleaser#1507] [Q&A] [answered]
We are trying a new website for goreleaser and we are looking for feedback!

Old site (react with 🎉): https://goreleaser.com
New site  (react with 🚀): https://goreleaser.now.sh/

React and/or let us know what you think!

PSA: Once we gather feedback and are "good" with it, we'll roll the domain change.

**Comment by crazy-max** [comment]
What I like with the new website:

* Auto TOC :star: :star:
* Search :star: :star: :star:
* Material design :star: :star: :star: :star:
* PR preview :star: :star: :star: :star: :star:

**Reply by caarlos0** [reply]
Same! The `Edit` button is pretty handy too :)

**Comment by caarlos0** [comment, marked as answer]
Just rolled it to the domain `goreleaser.com`

**Reply by caarlos0** [reply]
If you find any problems with the new website, let me know: open issues/PRs/etc :) 

marking this as done for now 🚀 

thanks everyon! 🤘

**Reply by crazy-max** [reply]
:rocket:
